src/app/tests/suites/certification/Test_TC_DRLK_2_13.yaml

```diff
@@ -0,0 +1,708 @@
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0e && DRLK.S.C28.Rsp
+      command: "SetAliroReaderConfig"
+      arguments:
+          values:
+              - name: "SigningKey"
+                value: signingkey
+              - name: "VerificationKey"
+                value: verificationkey
+              - name: "GroupIdentifier"
+                value: groupidentifier
+              - name: "GroupResolvingKey"
+                value: groupresolvingkey
+
+    - label:
+          "Step 12: TH sends SetUser Command to DUT to create an Aliro user with
+          the following values: OperationType as 0( DataOperationTypeEnum.Add),
+          UserIndex as 1, UserName as alirouser, UserUniqueID as 111, UserStatus
+          as 1(UserStatusEnum.OccupiedEnabled), UserType as
+          0(UserTypeEnum.UnrestrictedUser), CredentialRule as
+          0(CredentialRuleEnum.Single)"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F08 && DRLK.S.C1a.Rsp
+      command: "SetUser"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "OperationType"
+                value: 0
+              - name: "UserIndex"
+                value: 1
+              - name: "UserName"
+                value: alirouser
+              - name: "UserUniqueID"
+                value: 111
+              - name: "UserStatus"
+                value: 1
+              - name: "UserType"
+                value: 0
+              - name: "CredentialRule"
+                value: 0
+
+    - label:
+          "Step 13: TH sends SetCredential Command to DUT with CredentialType as
+          AliroCredentialIssuerKey and CredentialData as
+          alirocredentialissuerkey for the alirouser, OperationType as
+          0(DataOperationTypeEnum.Add), Credential as 6
+          1(CredentialType.AliroCredentialIssuerKey,CredentialIndex),
+          CredentialData as alirocredentialissuerkey, UserIndex as 1, UserStatus
+          as null, UserType as null"
+      PICS:
+          PICS_SKIP_SAMPLE_APP && DRLK.S.F0d && DRLK.S.C22.Rsp && DRLK.S.C23.Tx
+      command: "SetCredential"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "OperationType"
+                value: 0
+              - name: "Credential"
+                value: { CredentialType: 6, CredentialIndex: 1 }
+              - name: "CredentialData"
+                value: alirocredentialissuerkey
+              - name: "UserIndex"
+                value: 1
+              - name: "UserStatus"
+                value: null
+              - name: "UserType"
+                value: null
+      response:
+          values:
+              - name: "Status"
+                value: 0
+
+    #NEED TO CHECK WITH VALUES
+    - label: "Step 14: TH reads the LockUserChange event list from DUT"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0d || DRLK.S.E04
+      command: "readEvent"
+      event: "LockUserChange"
+      response:
+          value:
```

Comment:
This is not what the test plan says.  The test plan says there is either no event or an event like this, right?  Same for the other events.